crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -81,24 +80,24 @@ pub async fn create(
         write_account_to_accounts_file(account, accounts_file, chain_id, account_json.clone())?;
     }
 
-    if add_profile {
+    if add_profile.is_some() {
         let config = CastConfig {
             rpc_url: rpc_url.into(),
             account: account.into(),
             accounts_file: accounts_file.into(),
             keystore,
             ..Default::default()
         };
-        add_created_profile_to_configuration(&path_to_scarb_toml, &config)?;
+        add_created_profile_to_configuration(&add_profile, &config, &None)?;
     }
 
     Ok(AccountCreateResponse {
         address: Hex(address),
         max_fee: Decimal(max_fee),
-        add_profile: if add_profile {
-            "Profile successfully added to Scarb.toml".to_string()
+        add_profile: if add_profile.is_some() {
+            "Profile successfully added to snfoundry.toml".to_string()
```

Comment:
nit: The same comment as [here](https://github.com/foundry-rs/starknet-foundry/pull/1644/files#r1478354817)

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1644/commits/af08cbf36a17154306ea6f8cad673856ebb4cb20